core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -2260,4 +2260,8 @@ CacheConfigurationException storeConfiguredHasBothReadAndWriteOnly(String storeC
 
    @Message(value = "Invalid index startup mode: %s", id = 664)
    CacheConfigurationException invalidIndexStartUpMode(String invalidValue);
+
+   @LogMessage(level = ERROR)
+   @Message(value = "There was an error in submitted periodic task with %s, not rescheduling.", id = 665)
+   void scheduledTaskEncounteredThrowable(Object identifier, @Cause Throwable t);
```

Comment:
Should we log the exception here?

Reply:
This only happens if it fails and isn't retried, so I think we should log the exception as it will be the only time.